cpp/include/cugraph/graph_functions.hpp

```diff
@@ -423,15 +423,27 @@ decompress_to_edgelist(
 
 /**
  * @ingroup graph_functions_cpp
- * @brief Symmetrize edgelist.
+ * @brief Rule for combining edge properties when adding reciprocal edges.
+ */
+enum class edge_property_combining_rule_t {
+  PART_OF_UNIQUENESS,  /// This property is part of the uniqueness of the edge.
```

Comment:
Yeah... I am not sure about this... This is not really a combining rule...

And if this is selected for an edge property... how do we consider that in symmetrization.

Say edge ID is selected as "PART_OF_UNIQUENESS" property.

 In (src, dst, ID) triplets, how do we symmetrize (3, 5, 1) and (5, 3, 1) with reciprocal = true/false? What about (3, 5, 1) and (5, 3, 2)?
 
 I think we should make this cleaner and better document the expected behavior. Just reading the names, I can't properly guess how these functions will work.